Index: stage2/rootfs/etc/default/ifplugd
===================================================================
--- stage2.orig/rootfs/etc/default/ifplugd
+++ stage2/rootfs/etc/default/ifplugd
@@ -9,9 +9,10 @@
 # the udev script /lib/udev/ifplugd.agent to give default values.
 # The init script starts ifplugd for all interfaces listed in
 # INTERFACES, and the udev script starts ifplugd for all interfaces
 # listed in HOTPLUG_INTERFACES. The special value all starts one
 # ifplugd for all interfaces being present.
-INTERFACES=""
-HOTPLUG_INTERFACES=""
-ARGS="-q -f -u0 -d10 -w -I"
+
+INTERFACES="eth0"
+HOTPLUG_INTERFACES="auto"
+ARGS="-q -f -u0 -d5 -w -I"
 SUSPEND_ACTION="stop"
Index: stage2/rootfs/etc/ifplugd/ifplugd.action
===================================================================
--- stage2.orig/rootfs/etc/ifplugd/ifplugd.action
+++ stage2/rootfs/etc/ifplugd/ifplugd.action
@@ -23,10 +23,11 @@
 up)
 	run-parts --arg="$1" --arg="$2" /etc/ifplugd/action.d/
 	;;
 down)
 	run-parts --reverse --arg="$1" --arg="$2" /etc/ifplugd/action.d/
+	/sbin/dhclient -r $1
 	;;
 *)
 	echo "ifplugd.action: Incorrect action argument" >&2
 	exit 1
 	;;
